Software/Development/SemesterProject/App/analogAcquisitionManager.c

```diff
@@ -136,6 +136,94 @@ Uint16 getAnalogErrorStatus(void)
     return errorStatus;
 }
 
+/*
+ * Function to set thresholds for the current measurements.
+ * Absolute minimum and maximum is +/- 300A.
+ */
+void setCurrentThresholds(float *currentThresholdArray,
+                          float maximumCurrent, float minimumCurrent)
+{
+    float Rin = 9.1, biasVoltage = 0.817, opampGain = -1;

```

Comment:
Shouldn't this be constants saved in ROM?

Reply:
I think it would be nicer to have all the constant values in the same file constants.h in case we decide to change something eventually, we would know where to go.

Reply:
Same for the values of the resistances in the next functions

Reply:
I have moved all these to Constant.h